introduction.md

```diff
@@ -4,4 +4,11 @@ Iam bla bla bla bla bla.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?